regression/snapshot-harness/pointer-to-array-char/main.c

```diff
@@ -0,0 +1,29 @@
+#include <assert.h>
+#include <malloc.h>
+
+char *first;
+char *second;
+int array_size;
+
+void initialize()
+{
+  first = malloc(sizeof(char) * 12);
+  first = "1234567890a";
+  second = first;
+  array_size = 11;
+}
+
+void checkpoint()
+{
+}
+
+int main()
+{
+  initialize();
+  checkpoint();
+
+  assert(first == second);
+  assert(second[array_size - 1] == 'a');
+  //  assert(second[10]=='0');
```

Comment:
Is there a reason this assert is still commented out? If so, maybe worth including a comment explaining why.

Reply:
This one is redundant. Removed.